maven-parent/pom.xml

```diff
@@ -30,13 +30,16 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <ozone.artifactId>ozone</ozone.artifactId>
```

Comment:
I think one other place we want to use this is in the `Unpack Ozone` step, in the `includeArtifactIds`, so that we automatically unpack the artifact (it's probably fine to leave the directory things are unpacked into called "ozone".

Reply:
Thanks for pointing it out.
This is now fixed.

Regarding the directory names, I've updated them too so they use the provided artifact ID. That sounds much cleaner. I don't know if you think that could bring potential issues?

Reply:
No, I think that's much cleaner!